app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+    def search
+        @foods = FoodFacade.search_food(params[:q])
+        @search_count = FoodFacade.search_count(params[:q])
+    end
```

Comment:
Both of these methods call to the same Service endpoint. Can you think of a way to only make one api call and pull out the foods and the search count from just one api call?